TLM/TLM/ValueTuple.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace TrafficManager {
```

Comment:
Hmm why not in `\Util`?

Reply:
Because its so basic/core it could pollute all classes with \Util.
In .net its under the System namespace.